SS_benchfore.tpl

```diff
@@ -896,7 +898,7 @@ FUNCTION void Get_Benchmarks(const int show_MSY)
 
     Btgt_Fmult = F1(1);
     if (rundetail > 0 && mceval_counter == 0 && show_MSY == 1)
-      cout << " got_F0.1: " << Btgt_Fmult << endl;
+      echoinput << "Calculated F0.1: " << Btgt_Fmult << endl;
```

Comment:
Should this line have brackets?

Reply:
I don't see the "cout <<" statement in main. In this case, there should be brackets since there are two statements dependent on the condition.

Reply:
Sorry, in main it is "cout <<". In console cleanup, it is "echoinput <<".

Reply:
Single commands only, so brackets not required.